src/main/java/frc/robot/subsystems/ElevatorSubsystem.java

```diff
@@ -89,6 +64,24 @@ public void toL4() {
     elevatorPID.setSetpoint(ElevatorConstants.kElevatorStops[4]);
   }
 
+  public void toIntake() {
+    elevatorPID.setSetpoint(ElevatorConstants.kElevatorStops[5]);
+  }
+
+  // Logs print encoder and corresponding elevator height values
+  public void heightGet() {
```

Comment:
Just get it so that it returns a double, we can worry about printing to the dashboard in periodic. Check out Shuffleboard inputs